docs/organizations/managing-security-and-risk.md

```diff
@@ -25,7 +25,7 @@ Each panel shows the total count of matching items and contains a **Review** but
 
 When viewing the dashboard, you can:
 
--   Limit the total counts in each panel to a specific set of severities or repositories by clicking the **Severity** or **Repository** drop-downs above the main area.
+-   Limit the total counts in each panel to a specific set of severities or repositories by clicking the **Severity**, **Repository**, or **Security category** drop-downs above the main area.
```

Comment:
Small tweak, for completeness.

```suggestion
-   Limit the total counts in each panel to a specific set of severities, repositories, or categories by clicking the **Severity**, **Repository**, or **Security category** drop-downs above the main area.
```

Reply:
An alternative to avoid repetition:

```suggestion
-   Limit the total counts in each panel to a specific set of severities, repositories, or security categories by clicking filter drop-downs above the main area.
```

Reply:
LGTM. Added a final tweak for flow.

```suggestion
-   Limit the total counts in each panel to a specific set of severities, repositories, or security categories by clicking the filter drop-downs above the main area.
```